spec/index.bs

```diff
@@ -5245,7 +5245,7 @@ dictionary GPUVertexAttribute {
                 Otherwise:
                 - |attrib|.{{GPUVertexAttribute/offset}} + sizeof(|attrib|.{{GPUVertexAttribute/format}}) &le;
                     |descriptor|.{{GPUVertexBufferLayout/arrayStride}}.
-            - |attrib|.{{GPUVertexAttribute/offset}} is a multiple of the size of one component of
+            - |attrib|.{{GPUVertexAttribute/offset}} is a multiple of the minimum of 4 and the size of
```

Comment:
in the line above we use `sizeof(|attrib|.{{GPUVertexAttribute/format}})`, let's be consistent